app/controllers/games/list.js

```diff
@@ -6,5 +6,10 @@ export default Ember.Controller.extend({
     return !Ember.isNone(this.get('session.user'));
   }.property('session.user'),
 
-  cannotCreateGame: Ember.computed.not('canCreateGame')
+  cannotCreateGame: Ember.computed.not('canCreateGame'),
+
+  /** @property {Array<Game>} all the games that haven't been ended */
+  joinableGames: Ember.computed('model.[].status', function () {
```

Comment:
...I misread somewhere a long time ago that @each and [] were aliases of each other. good lord i've been doing that wrong for quite some time now but never experiencing problems...i guess now that makes sense that binding to [] worked because i often returned a brand new array. i'll fix this.